program/program/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿bool sucsessX;
+bool sucsessY;
+string strNumX;
+string strNumY;
+int sum = 0;
+int X;
+int Y;
+
+
+
+do
+{
+    Console.Write("Write X: ");
+    strNumX = Console.ReadLine();
+    sucsessX = int.TryParse(strNumX, out int StrNumX);
+    Console.WriteLine(sucsessX ? $"It is number {StrNumX}" : "It is not number");
+} while (!sucsessX);
+
+do
+{
+    Console.Write("Write Y: ");
+    strNumY = Console.ReadLine();
+    sucsessY = int.TryParse(strNumY, out int StrNumY);
+    Console.WriteLine(sucsessY ? $"It is number {StrNumY}" : "It is not number");
+
+} while (!sucsessY);
+
+X = Convert.ToInt32(strNumX);
+Y = Convert.ToInt32(strNumY);
+if (X < Y)
```

Comment:
you can skip this condition, if X==Y then for will run only once and add to sum only X